LICENSE

```diff
@@ -1,14 +1,191 @@
-            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
```

Comment:
What happened here, why is the license Apache one now, and Copyrighted with your surname and Akamai email :scream:

Reply:
well, I am gonna change email, but this license stays since it's public boi

Reply:
and it's mine, MINE, ALL MINE!